lib/api/asset/v1/service.test.ts

```diff
@@ -96,3 +96,50 @@ test("useAsset: the API is called with the right parameters", async () => {
   );
   expect(response).toBe(returnedValue);
 });
+
+test("createAsset: the API is called with the right parameters", async () => {
```

Comment:
This doesn't describe a test. This states an aspiration. Rename it using a valid testing methodology (e.g. If / When / Then)

Reply:
Added item no. 44 in tech debt spreadsheet
https://docs.google.com/spreadsheets/d/1TUS7XjtLX6cpdN_r4Spg2Lw12B4921q004eJKCEJCHQ/edit#gid=0